pkg/plugin/installer/installer.go

```diff
@@ -64,19 +64,19 @@ func Update(i Installer) error {
 }
 
 // NewForSource determines the correct Installer for the given source.
-func NewForSource(source, version string) (Installer, error) {
+func NewForSource(source, version, pluginsDirectory string) (Installer, error) {
```

Comment:
So unfortunately, this would constitute a breaking change as `installer.NewForSource` is a publicly exposed function call. Is there another way we can fix this?

For reference: https://pkg.go.dev/helm.sh/helm/v3@v3.6.3/pkg/plugin/installer#NewForSource

Reply:
Wait for me to think. Do you have any suggestions?